code/scenes/src/main/kotlin/org/fbme/scenes/controllers/diagram/DiagramController.kt

```diff
@@ -14,4 +16,5 @@ interface DiagramController<CompT, PortT, ConnT> {
     fun setTarget(edge: ConnT, port: PortT)
     fun removeEdge(edge: ConnT)
     fun addEdge(sourcePort: PortT, targetPort: PortT): ConnT?
+    fun addPort(port: PortT)
```

Comment:
why component parameter not passed here?

Reply:
Don't get what mean here

Reply:
Done